test/unit/parachain/loans.test.ts

```diff
@@ -128,4 +134,173 @@ describe("DefaultLoansAPI", () => {
+                collateralThresholdRelayCurrency,
+                exchangeRateRelayCurrency
+            ),
+            [wrappedCurrency.ticker]: mockLoanAsset(
+                wrappedCurrency,
+                liquidationThresholdWrappedCurrency,
+                collateralThresholdWrappedCurrency,
+                exchangeRateWrappedCurrency
+            ),
+        };
+
+        it("should return correct amounts in BTC", () => {
+            const governanceCurrencyLentAmount = Big(100);
+            const relayCurrencyBorrowedAmount = Big(1);
+            const lendPositions = [mockLendPosition(testGovernanceCurrency, governanceCurrencyLentAmount)];
+            const borrowPositions = [mockBorrowPosition(testRelayCurrency, relayCurrencyBorrowedAmount)];
+
+            const expectedTotalLentBtc = governanceCurrencyLentAmount.div(exchangeRateGovernanceCurrency);
+            const expectedTotalBorrowedBtc = relayCurrencyBorrowedAmount.div(exchangeRateRelayCurrency);
+            const expectedTotalCollateralBtc = expectedTotalLentBtc;
+            const expectedBorrowLimitBtc = expectedTotalCollateralBtc
+                .mul(collateralThresholdGovernanceCurrency)
+                .sub(expectedTotalBorrowedBtc);
+
+            const { totalLentBtc, totalBorrowedBtc, totalCollateralBtc, borrowLimitBtc } =
+                loansApi.getLoanCollateralInfo(lendPositions, borrowPositions, loanAssets);
+
+            expect(
+                totalLentBtc.toBig().eq(expectedTotalLentBtc),
+                `Total lent amount: ${totalLentBtc
+                    .toBig()
+                    .toString()} doesn't match expected amount ${expectedTotalLentBtc.toString()}`
+            ).to.be.true;
+            expect(
+                totalBorrowedBtc.toBig().eq(expectedTotalBorrowedBtc),
+                `Total borrowed amount: ${totalBorrowedBtc.toString()} doesn't match expected amount ${expectedTotalBorrowedBtc.toString()}`
+            ).to.be.true;
+            expect(
+                totalCollateralBtc.toBig().eq(expectedTotalCollateralBtc),
+                `Collateral amount: ${totalCollateralBtc.toString()} doesn't match expected amount ${expectedTotalCollateralBtc.toString()}`
+            ).to.be.true;
+            expect(
+                borrowLimitBtc.toBig().eq(expectedBorrowLimitBtc),
+                `Borrow limit amount: ${borrowLimitBtc.toString()} doesn't match expected amount ${expectedBorrowLimitBtc.toString()}`
+            ).to.be.true;
+        });
+
+        it("should compute correct LTV and average thresholds", () => {
+            const governanceCurrencyLentAmount = Big(100);
+            const wrappedCurrencyLentAmount = Big(2);
+            const relayCurrencyLentAmount = Big(1000); // Should not affect the computation since it won't be enabled as collateral.
+
+            const governanceCurrencyBorrowedAmount = Big(10);
+            const relayCurrencyBorrowedAmount = Big(10);
```

Comment:
Here too, please use `MonetaryAmount` instead of `Big`. The signatures of `mockLendPosition` and `mockBorrowPosition` should be easy to change so you don't have to unroll the object when calling the functions